test/testdata.go

```diff
@@ -0,0 +1,3 @@
+package test
```

Comment:
folder: faker
file: bitfaker/bitcoin.go

use `faker.BitcoinTestAddress`

Reply:
fixed the point